server/models/Dog.model.js

```diff
@@ -0,0 +1,46 @@
+const mongoose = require("mongoose");
+const currentYear = new Date().getFullYear();
+
+const dogSchema = new mongoose.Schema(
+  {
+    user: { type: mongoose.Types.ObjectId, required: true, ref: "user" },
+    name: { type: String, required: true, trim: true, maxlength: 50 },
+    breed: { type: String, required: true, trim: true, maxlength: 50 },
+    size: {
+      // [extra small | small | meduim | large | giant]
```

Comment:
This is conveyed clearly thanks to the `enum` option.

Reply:
yes it is :)